{"uid":"67fa45c54549fd1e","name":"test_enable_user_with_unregistered_email","fullName":"tests.authentication.test_enable_user#test_enable_user_with_unregistered_email","historyId":"03a9242f839685d9f7956365b3ba86f7","time":{"start":1729851733477,"stop":1729851735291,"duration":1814},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError: Expected status code 404 but got 500. Error message: El usuario no esta registrado","statusTrace":"get_headers = <function get_headers.<locals>._get_headers at 0x7f35a0ebcfe0>\nenable_user_payload = {'email': 'unregistered@example.com'}\n\n    @allure.suite('Enable User')\n    @allure.epic('Authentication')\n    @allure.feature('Enable User')\n    @allure.story('Enable User with Unregistered Email')\n    @allure.tag('author: Jeyson')\n    @pytest.mark.functional\n    @pytest.mark.negative\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\",condition=True)\n    def test_enable_user_with_unregistered_email(get_headers, enable_user_payload):\n        url = AuthenticationEndpoints.enable_user()\n        headers = Auth().auth_valid_credential(get_headers)\n        enable_user_payload['email'] = \"unregistered@example.com\"\n        response = RealHomeRequest.post(url, headers, enable_user_payload)\n    \n>       assert_status_code_not_found(response)\n\ntests/authentication/test_enable_user.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [500]>\n\n    def assert_status_code_not_found(response):\n>       assert response.status_code == 404, f\"Expected status code 404 but got {response.status_code}. Error message: {extract_error_message(response)}\"\nE       AssertionError: Expected status code 404 but got 500. Error message: El usuario no esta registrado\n\ncore/assertions/status_code.py:8: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"enable_user_payload","time":{"start":1729851726096,"stop":1729851726096,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"get_headers","time":{"start":1729851733476,"stop":1729851733476,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"tag","value":"author: Jeyson"},{"name":"feature","value":"Enable User"},{"name":"suite","value":"Enable User"},{"name":"epic","value":"Authentication"},{"name":"story","value":"Enable User with Unregistered Email"},{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"negative"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.authentication"},{"name":"host","value":"fv-az841-46"},{"name":"thread","value":"2144-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.authentication.test_enable_user"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["negative","functional","author: Jeyson","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"67fa45c54549fd1e.json","parameterValues":[]}